packages/application-shell/src/components/navbar/use-navbar-state-manager.ts

```diff
@@ -21,6 +21,15 @@ import useApplicationsMenu from '../../hooks/use-applications-menu';
 import FetchProjectExtensionsNavbar from './fetch-project-extensions-navbar.settings.graphql';
 import nonNullable from './non-nullable';
 
+enum ActionTypes {
```

Comment:
Nit: Prefix with T. We somewhat settled on it in app-kit so that types are easily distinguishable from anything else.

Reply:
we don't do that for everything else in this file. May I keep it this way?